bigtop-packages/src/charm/kafka/layer-kafka/tests/01-deploy.py

```diff
@@ -33,9 +33,19 @@ def setUpClass(cls):
         cls.d.configure('openjdk', {'java-type': 'jdk',
                                     'java-major': '8'})
 
-        cls.d.relate('kafka:zookeeper', 'zk:zkclient')
+        try:
```

Comment:
+1 to renaming the relation to be consistent.  We haven't promulgated that yet, have we?